api/Database/Models/TagAnalysis.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+#pragma warning disable CS8618
+namespace api.Database.Models;
+
+public class TagAnalysis
+{
+    [Key]
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public string Id { get; set; }
+
+    [Required]
+    public string Tag { get; set; }
+
+    [Required]
+    public List<AnalysisType> AnalysisToBeRun { get; set; } = [];
```

Comment:
use plural term of analysis: analyses
```suggestion
    public List<AnalysisType> AnalysesToBeRun { get; set; } = [];
```